index.html

```diff
@@ -11,6 +11,7 @@ <h1>Hello There Syntax Squad!</h1>
       <p>We would like you all to tryout GitHub Flow.</p>
       <p>Winnie and Jules 💋</p>
       <p>Welcome Ayo, Kelsey, Jose and Albert</p>
+      <p>Kelsey's practice 🌻</p>
```

Comment:
Cool! you might want to try change up something and tick off comments. I'll request change.

Reply:
Hi Kelsey can you add me as a reviewer so I can practise that side of the flow please :)

Reply:
Hey @josetutis I think I've made you a reviewer now, any issues let me know :)